operations/kubernetes/Restore_Postgres.md

```diff
@@ -26,8 +26,8 @@ In the event that the Spire Postgres cluster must be rebuilt and the data restor
 
     * If a manual dump of the database was taken, then check that the dump file exists in a location off the Postgres cluster. It will be needed in the steps below.
     * If the database is being automatically backed up, then the most recent version of the dump and the secrets should exist in the `postgres-backup` S3 bucket.
-    These will be needed in the steps below. List the files in the `postgres-backup` S3 bucket and if the files exist, download the dump and secrets out of the S3 bucket.
-    The `cray artifacts` CLI can be used list and download the files. Note that the `.psql` file contains the database dump and the .manifest file contains the secrets.
+      These will be needed in the steps below. List the files in the `postgres-backup` S3 bucket and if the files exist, download the dump and secrets out of the S3 bucket.
+      The `cray artifacts` CLI can be used list and download the files. Note that the `.psql` file contains the database dump and the `.manifest` file contains the secrets.
```

Comment:
"CLI can be used to list ..."
The "to" is missing here and in a few other places below.

Reply:
Good catch. Fixed.